src/amp-story-player/amp-story-component-manager.js

```diff
@@ -43,19 +55,39 @@ export class AmpStoryComponentManager {
       return;
     }
 
-    const intersectingCallback = (entries) => {
+    const ioPrerenderCb = (entries) => {
+      entries.forEach((entry) => {
+        if (!entry.isIntersecting) {
+          return;
+        }
+        elImpl.prerenderCallback();
+
+        prerenderObserver.unobserve(elImpl.getElement());
+      });
+    };
+
+    const rootMarginInPercentage =
+      (MIN_VIEWPORT_PRERENDER_DISTANCE * 100).toString() + '%';
```

Comment:
Template string might have the same effect here: `${MIN_VIEWPORT_PRERENDER_DISTANCE * 100}%`